<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitoring_edit</class>
 <widget class="QDialog" name="Monitoring_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(203, 211, 255);
color:black;</string>
   </property>
   <property name="text">
    <string>Saved</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(203, 211, 255);
color:black;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>258</width>
     <height>242</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="Month_combo">
        <item>
         <property name="text">
          <string>Jan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Feb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>May</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jun</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sep</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oct</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nov</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dec</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Double click Key section you want to edit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Year_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Month_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Day_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cate_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>155</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>43</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amount_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Desc_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>561</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(243, 243, 255)</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_7</zorder>
  <zorder>Year_edit</zorder>
  <zorder>Month_edit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>Day_edit</zorder>
  <zorder>label_4</zorder>
  <zorder>Cate_edit</zorder>
  <zorder>label_5</zorder>
  <zorder>Amount_edit</zorder>
  <zorder>label_6</zorder>
  <zorder>Desc_edit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
